plutus-core/plutus-core/src/PlutusCore/DeBruijn.hs

```diff
@@ -6,10 +6,14 @@
 -- | Support for using de Bruijn indices for term and type names.
 module PlutusCore.DeBruijn
     ( Index (..)
+    , Level (..)
+    , LevelInfo (..)
     , HasIndex (..)
     , DeBruijn (..)
     , NamedDeBruijn (..)
-    , FakeNamedDeBruijn (..)
+    -- we follow the same approach as Renamed, expose the constructor from Internal module,
+    -- but hide it in this parent module.
+    , FakeNamedDeBruijn (unFakeNamedDeBruijn)
```

Comment:
actually exposing the field is unsafe because it can be used as a constructor, but we can address this (and other such instances) in another PR